### PR TITLE
Last bits to make Lucet publishable on crates.io

### PR DESCRIPTION
**Pull request by jedisct1**
Some required changes to make Lucet crates something we can publish.

### CODE REVIEW

File: lucet-wasi/build.rs [hunk 1]
```diff
@@ -32,6 +34,10 @@ fn wasm_clang_root() -> PathBuf {
     }
 }
 
+#[cfg(not(feature = "bindgen"))]
```
Comment (pchickey) [inline lucet-wasi/build.rs]: Can you add a comment explaining why this exists?

File: lucet-wasi/Cargo.toml [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
-lucetc = { path = "../lucetc" }
+lucet-wasi-sdk = { path = "../lucet-wasi-sdk", version = "0.1.1" }
+lucetc = { path = "../lucetc", version = "0.1.1" }
 tempfile = "3.0"
 
-[build-dependencies]
-bindgen = "0.47"
+[build-dependencies.bindgen]
+version = "0.47"
+optional = true
```
Comment (pchickey) [inline lucet-wasi/Cargo.toml]: add a brief comment here as well if you don't mind

File: helpers/bump-global-version.sh [hunk 3]
```diff
@@ -6,7 +6,7 @@ VERSION=$(grep '#*Lucet version ' Cargo.toml | sed 's/^ *# Lucet version *//')
 dry_run() {
     echo "Checking if the package can be published (dry run)..."
     echo
-    find lucetc lucet-* benchmarks/lucet-* -type f -maxdepth 1 -name 'Cargo.toml' -print | while read -r file; do
+    find lucetc lucet-* benchmarks/lucet-* lucet-runtime/lucet-*-type f -maxdepth 1 -name 'Cargo.toml' -print | while read -r file; do
```
Comment (pchickey) [inline helpers/bump-global-version.sh]: There is also a crate at lucet-idl/lucet-idl-test that we'd want to include.

Also, is there a space missing before `-type`?

Comment (jedisct1) [reply helpers/bump-global-version.sh]: Indeed :)

File: lucet-wasi/Cargo.toml (hunk 2 above)
Comment (jedisct1) [reply lucet-wasi/Cargo.toml]: Done.

File: lucet-wasi/build.rs (hunk 1 above)
Comment (jedisct1) [reply lucet-wasi/build.rs]: Done.

**Pull request merged**